lighthouse-core/computed/metrics/cumulative-layout-shift.js

```diff
@@ -37,15 +36,6 @@ class CumulativeLayoutShift {
         continue;
       }
 
-      // For all-frames CLS calculation, we rely on `weighted_score_delta`, which
-      // was added in Chrome 90: https://crbug.com/1173139
-      if (event.args.data.weighted_score_delta === undefined) {
-        throw new LHError(
-          LHError.errors.UNSUPPORTED_OLD_CHROME,
-          {featureName: 'Cumulative Layout Shift'}
-        );
-      }
```

Comment:
+1 to not defaulting to 0. It shouldn't ever be omitted without a regression in Chrome ([always included here](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/layout/layout_shift_tracker.cc;l=743;drc=b908cfa41e42bd30478334146167ffea418b7236)), so I'd be good with throwing just a regular error as @adamraine suggested, or it could get a `continue` like the `event.args.data.is_main_frame === undefined` check above does (which has a similar history to this check).

Reply:
Instead of moving this to another error thats less specific, wouldn't keeping the current `UNSUPPORTED_OLD_CHROME` error be the best course of action then? Switching to `continue` would still end up calculating resulting CLS as 0, when `weighted_score_delta` is missing (or as 1, for the 10x10px iframe case mentioned [here](https://github.com/GoogleChrome/lighthouse/issues/12023))?

Reply:
I would prefer a generic error, since this seems more like a type narrowing case at this point.